README.md

```diff
@@ -23,47 +15,72 @@ We also [provide](https://github.com/moabitcoin/ig65m-pytorch/releases) converte
 |  R(2+1)D_34   | IG-65M + None       | 32x112x112 | [*r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth)                                               | [*r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb)                                            |
 |  R(2+1)D_34   | IG-65M + Kinetics  | 32x112x112 | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth) | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb)  |
 
+Notes:
+- ONNX models provided here have not been optimized for inference.
+- For models fine-tuned on Kinetics you can use the labels from [here](https://github.com/Showmax/kinetics-downloader/blob/68bd8bc3b9e30da83db9e34cb7d867dcda705cb4/resources/classes.json)
+
+
+## Usage :computer:
 
-### Convert model :spaghetti:
+The following describes how to use the model in your own project and how to use our conversion and extraction tools.
+
+### In Your Own Project
+
+- See `convert.py` and copy the `r2plus1d_34` model architecture definition
+- See `exract.py` for how to load the corresponding weights into the model
+
+Note: we require torchvision v0.4 or later for the model architecture building blocks
+
+### Development and Tools
+
+We provide CPU and [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) based GPU Dockerfiles for self-contained and reproducible environments.
+
+Use the convenience Makefile to build the Docker image and then get into the container mounting a host directory to `/data` inside the container:
+
+```
+make
+make run datadir=/Path/To/My/Videos
 ```
-python convert.py --help
-convert.py [-h] --frames {8,32} --classes {400,487} pkl out
 
-positional arguments:
-  pkl                  .pkl file to read the R(2+1)D 34 layer weights from
-  out                  prefix to save converted R(2+1)D 34 layer weights to
+By default we build and run the CPU Docker images; for GPUs run:
 
-optional arguments:
-  -h, --help           show this help message and exit
-  --frames {8,32}      clip frames for video model
-  --classes {400,487}  classes in last layer
```

Comment:
I think we should explain what each of the numbers mean. 400 for kinetics, 487 for T=8 and 259 for T=32 ?

Reply:
Yes, adding